### DISCUSSION TITLE
昨日负载过高，已关闭新隧道创建

### DISCUSSION THREAD

**Discussion by Tonemoretime** [natfrp/wiki#97] [使用问题] [answered]
这样显示的隧道一般第二天是不是会显示正常可以创建？还是会一直等好久好久。。。。。。

**Comment by fakeboboliu** [comment, marked as answer]
直到负载降低到我们认为不会影响用户体验，对应节点才会被重新开放，在这之前请使用其他节点。

不过今天看起来已经都开放了。
